api/http.go

```diff
@@ -62,8 +62,12 @@ func NewCatalystAPIRouter(cli config.Cli, vodEngine *pipeline.Coordinator, bal b
 		Config:   cli,
 		Lapi:     lapi,
 	}
+	analyticsApiHandlers := &handlers.AnalyticsHandlersCollection{}
```

Comment:
Could this go down inside the conditional too?

Reply:
Good spot, updated in 966e638a1cc48551294ffc960a8f82364098cb9c